backend/src/main/java/team/teamby/teambyteam/feed/application/FeedThreadService.java

```diff
@@ -37,4 +51,40 @@ public Long write(
         final Feed savedFeed = feedRepository.save(feed);
         return savedFeed.getId();
     }
+
+    @Transactional(readOnly = true)
+    public FeedsResponse firstRead(final Long teamPlaceId, final Integer size) {
+        final Pageable pageable = getPageableInitSize(size);
+        final List<Feed> list = feedRepository.findByTeamPlaceId(teamPlaceId, pageable);
+        final List<FeedResponse> feedResponses = mapFeedResponses(list);
+
+        return FeedsResponse.of(feedResponses);
+    }
+
+    @Transactional(readOnly = true)
+    public FeedsResponse reRead(final Long teamPlaceId, final Long threadId, final Integer size) {
+        final Pageable pageable = getPageableInitSize(size);
+        final List<Feed> list = feedRepository.findByTeamPlaceIdAndIdLessThan(teamPlaceId, threadId, pageable);
+        final List<FeedResponse> feedResponses = mapFeedResponses(list);
+
+        return FeedsResponse.of(feedResponses);
+    }
+
+    private Pageable getPageableInitSize(final Integer size) {
+        return PageRequest.of(FIRST_PAGE, size, SORT_DIRECTION, SORT_CRITERIA);
+    }
+
+    private List<FeedResponse> mapFeedResponses(final List<Feed> feeds) {
+        return feeds.stream().map(feed -> {
+            if (FeedType.THREAD == feed.getType()) {
+                final Optional<Member> member = memberRepository.findById(feed.getAuthorId());
+                final Member member1 = member.orElseThrow(MemberException.MemberNotFoundException::new);
```

Comment:
```suggestion
                final Member member = memberRepository.findById(feed.getAuthorId())
                        .orElseThrow(MemberException.MemberNotFoundException::new);
```

위와같이 하면 member1같은게 안생기지 않을까요...?

Reply:
적용 완료 했습니다.